### PR TITLE
Move welcome pages to react

### LINKED ISSUE #136: Change welcome pages to use react components
In core we've merged https://github.com/theforeman/foreman/pull/8418
It transfer all welcome pages to React, but we already have the old versions extracted here.

### PR DESCRIPTION
**Pull request by nadjaheitmann**
Fixes #136 

@ezr-ondrej There will be a PR that removes the react components from your puppet_extraction branch. I had to remove them for testing.

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
PR against my puppet_extraction branch is the way to go now, until that is merged in Foreman core.

**Comment by nadjaheitmann** [comment]
> PR against my puppet_extraction branch is the way to go now, until that is merged in Foreman core.

Managed to open a PR against your branch: https://github.com/ezr-ondrej/foreman/pull/67

File: .github/workflows/ruby_tests.yml [hunk 1]
```diff
@@ -60,6 +60,7 @@ jobs:
         run: |
           bundle exec rake app:db:create
           bundle exec rake app:db:migrate
+          bundle exec rake app:webpack:compile
```
Comment (ezr-ondrej) [inline .github/workflows/ruby_tests.yml:63]: I'd probably rather drop the specs for the React pages. There is not much need, I just liked to have some kind of testing for them. But with JS testing, that would be redundant.

**Comment by ezr-ondrej** [comment]
@nadjaheitmann Please don't try to make webpack work in the specs, it's a lot of works and I don't think we want that.
Just drop the welcome template specs, as they are no longer needed, we have tests for the components :)

File: .github/workflows/ruby_tests.yml (hunk 1 above)
Comment (nadjaheitmann) [reply .github/workflows/ruby_tests.yml]: Done :)

**Comment by nadjaheitmann** [comment]
@ezr-ondrej Anything else I should add here? As you have already merged the corresponding PR in the `puppet_extraction` branch, it makes sense to merge this as well. Otherwise there should be an error on the environments welcome page with Puppet plugin master and newest `puppet_extraction` branch.

**Comment by ezr-ondrej** [comment]
Sorry, I've forgotten to push the button here :)

**Review by ezr-ondrej — APPROVED** [review]
Thanks @nadjaheitmann ! :+1:

**Pull request merged**